journal/modernized/08-chapter-8.adoc

```diff
@@ -403,7 +403,7 @@ especially as he is declared to be worse than an infidel who provides not for hi
 Then mourning over those who were not sensible of their situation,
 and of the great salvation offered to all, he said,
 "`I now again tell you what I have so often declared
-that 'other foundation can no man lay than is laid,
+that no other foundation can man lay than is laid,
```

Comment:
Moving the `no` seems correct here, but I think we should probably put the apostrophe back, since it's a quote within a quote.  So the line should read:

```
that 'no other foundation can man lay than is laid,
```